source/blog/articles/2019-07-the-last-stretch.html.markdown

```diff
@@ -0,0 +1,49 @@
+
+The second day of our **literature blitz**, as we call them, also went very well. Cornelius and I did have a lady run us off, but that is not uncommon when we do literature distribution. We just go to the next entrance out of sight and hope nobody will see us as threats! The total amount of *Bible First* invitations we distributed during the two days in Lutsk was 33,000.
+
+On Friday, we started by having devotions led by Cornelius, followed by book reports, which was altogether very interesting and encouraging. Joshua went to an automotive electrician and got the van fixed quickly, praise the Lord. The problem was a relay that was corroded on the inside. So we were able to get the gear in the van and go downtown to do surveys and street preaching. Several in the team had good conversations with people there. Then we set up the gear for street preaching, and Josh went first. As he was finishing up, we started having some rain. We waited it out and Nelson went on to preach, though he had to finish his message in the rain. Since it was pouring more and more and the street got mostly emptied from its passer-bys, we decided we would have Cornelius preach on another day.
+
+We did our last day of literature distribution on Saturday, where we distributed 13,375 invites in three smaller cities: Truskavets, Stebnyk, and Boryslav. Everything went smoothly, and I even got to do the very last set of P.O. boxes of CMO!
+
+[comment]: <> (insert picture of me doing the last pid'yeezd)
+
+We were able to go to church on Sunday, and enjoyed nice Ukrainian hymns played by the church musicians. We then drove to Yasenytsya-Zamkova, where our last film showing was. People started showing up as early as thirty minutes in advance, which is rather uncommon. By 17:00, when the movie started, we already had a significant crowd, peaking to twenty-three during the movie. Just as the movie began playing, we had a good rain outside, meaning the attendants were not in a hurry to leave the hall! The crowd was quite attentive, and even clapped after Josh did his Gospel presentation and outro message.
+
+Once Josh had finished speaking, a man from the audience stood up and introduced himself as a regional government official. He was very complimentary of our presentation, and thanked us for our work. He then introduced a colleague of his who is running for parliament. This man was also very positive about the film showing. He spoke for several minutes about his campaign for parliament, emphasizing the importance of a strong family unit. At the end, we prayed for both men, noting the passage in Scripture which instructs us to pray for those in positions of authority.
```

Comment:
Perfect! You're the one that understood was what said in that conversation, so thanks for telling the story!